constants_local.xml

```diff
@@ -295,8 +296,5 @@
         <SHARDPUBKEY>02F650040BC0F3158B5D1A870EFC0ABE84FF02A4021A222FC49F9AD070ACFA2DA7</SHARDPUBKEY>
         <SHARDPUBKEY>030DFD4A0CFD68016DF6EFC4A6BF0B67BA42ABEAF8D9AD65C25E97B9DA90CA4DB1</SHARDPUBKEY>
         <SHARDPUBKEY>03129EAEF8A136355FB1485941A593B4BEC4DFC5504D1114138A6D92332005DC59</SHARDPUBKEY>
-        <SHARDPUBKEY>037D49C420B04B9862BC1F0660544FF27F1D81EA0E5E7C161F1647FBF239F8780E</SHARDPUBKEY>
```

Comment:
Just want to confirm if we need to remove these keys

Reply:
Yes. It is or local test. This is so that the local node is no longer a shard guard.